KarhooUISDK/CommonUI/Views/KarhooNewTextField.swift

```diff
@@ -0,0 +1,86 @@
+//
+//  KarhooNewTextField.swift
+//  KarhooUISDK
+//
+//  Created by Bartlomiej Sopala on 16/01/2023.
+//  Copyright © 2023 Flit Technologies Ltd. All rights reserved.
+//
+
+import SwiftUI
+
+struct KarhooNewTextField: View {
+   
+    @Binding var textFieldText: String
+    @Binding var isTextfieldValid: Bool
+    @State private var isFirstResponder: Bool = false
+
+    var placeholder: String
+    var errorMessage: String
+    var contentType: KarhooTextInputViewContentType
+    
+    
+    var body: some View {
+        VStack(alignment: .leading, spacing: 0) {
+            HStack{
+                TextField(placeholder, text: $textFieldText, onEditingChanged: { (editingChanged) in
+                    if editingChanged {
+                        isFirstResponder = true
+                    } else {
+                        isFirstResponder = false
+                    }
+                })
+                .onChange(of: textFieldText) { newValue in
+                    withAnimation {
+                        isTextfieldValid = getTextFieldValidity(newValue: newValue)
+                    }
+                }
+                Button {
+                    textFieldText = ""
+                } label: {
+                    Image(uiImage: .uisdkImage("kh_uisdk_cross_in_circle"))
```

Comment:
I think `.resisable()` needs to be called to make image change it's size, not only wrapper size.